### PR TITLE
update mir examples

### PR DESCRIPTION
**Pull request by 9il**
Not all the new links working now, but they will work after a few days.

### CODE REVIEW

**Comment by ZombineDev** [comment]
@9il is this PR ready, or you prefer to wait for the links to become available?

File: dub/mir.md [hunk 1]
```diff
@@ -5,27 +5,38 @@ The package includes:
  - [mir-algorithm package](dub/mir-algorithm). Dlang core library for math, finance and a home for Dlang multidimensional array package - ndslice.
  - [mir-random package](dub/mir-random). Advanced random number generators.
  - Sparse tensors
- - Combinatorics
```
Comment (ZombineDev) [inline dub/mir.md]: BTW, what happened with the combinatorics? Were they moved to a separate repo, or you plan to deprecate them?

File: dub/lubeck.md [hunk 2]
```diff
@@ -11,20 +11,20 @@ backends are recommeded for Linux and Windows.
 ## Links
 
  - [GitHub](https://github.com/kaleidicassociates/lubeck)
- - [Mir Algorithm API](http://docs.algorithm.dlang.io)
- - [Mir Random API](http://docs.algorithm.dlang.io)
- - [Mir API](http://docs.mir.dlang.io)
+ - [Mir Algorithm API](http://mir-algorithm.libmir.org)
```
Comment (ZombineDev) [inline dub/lubeck.md]: I strongly suggest using `https://` for all links (if possible).

Comment (ZombineDev) [reply dub/lubeck.md]: For static websites, you can try the free version of Netlify - I am quite happy with it. Alternatively, if you want keep your current setup and want an easy way to enable HTTPS throughout, you can setup Cloudflare.

File: dub/mir.md (hunk 1 above)
Comment (9il) [reply dub/mir.md]: It is a part of mir-algorithm now

**Pull request merged**